content/docs/09-registries-and-packs/3-add-custom-packs.md

```diff
@@ -35,7 +35,7 @@ Each pack contains a metadata file named `pack.json`. The table below explains i
 | layer | String | True | Relevant layer that this pack should be part of; such as os, k8s, cni, csi, addon |
 | addonType | String | False | Addon-type must be set for packs that have the layer set to Addon. The value must be one of the following: logging, monitoring, load balancer, authentication, ingress, security. Setting a relevant correct addon type ensures packs are organized correctly on the management console making it easy for profile authors to find packs. |
 | version | String | True | A Semantic version for the pack. It is recommended that the pack version be the same as the underlying integration it is being created for. For example, the version for the pack that will install Prometheus 2.3.4, should set to 2.3.4. |
-| cloudTypes | Array | True | Supported cloud types are AWS, Azure, VMware. One or more types can be provided for a pack. |
+| cloudTypes | Array | True | Supported cloud types are AWS, Azure, GCP, and VMware. One or more types can be provided for a pack. |
```

Comment:
we need to add  openstack, maas and Tencent

Reply:
Actually here is the full list.  Sorry its a bit confusing with the description first and actual property value second

AWS - aws
AWS EKS - eks
Google - gcp
Azure - azure
Azure Managed Kubernetes Service- aks
VMware -vpehere
OpenStack - openstack
Tencent TKE -tke
MaaS -maas
Virtual Edge and virtual clusters - libvirt
Edge 1.0 containerized edge - edge
Edge 2.0 - edge-native 
CoxEdge -coxedge
Any of the supported cloud types -all

Reply:
Ideally we should say supported values are then have a table with the valid property values and the descriptions.  I am assuming we are a case sensitive platform so AWS isn't the same as AWS. Also today it says VMware, but the actually property value you need to specify is vsphere for example

Reply:
@liltigermel  
Added the above values to the cloudType description.

Reply:
@liltigermel Added the above values to cloudType description.